.github/workflows/ci_cd.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest]
-        python-version: ['3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.9', '3.10', '3.11', '3.12']
```

Comment:
Just noting that PyFl has 3.9 for the main version and here we have 3.10 (MAIN_PYTHON_VERSION: '3.10'). Is that deliberate in some way? Shall we align them?

Reply:
It is 3.9 in all other repos as well. I'm also not sure if it is deliberate.

Reply:
@hpohekar and @mkundu1, any thoughts if the python version can be aligned?

Reply:
Maybe it was added by the template. But it is safer to run the smoke test with minimum supported Python version to catch any issue due to unavailable language features. Let's change it to 3.9.

Reply:
Done, Thank you.